Services/ProjectService/appsettings.json

```diff
@@ -10,6 +10,13 @@
   "ConnectionStrings": {
     "SQLConnectionString": "Server=localhost;Database=ProjectDB;Trusted_Connection=True;"
   },
+  "RabbitMQ": {
+    "Host": "localhost",
+    "Username": "ProjectService",
+    "Password": "E68A3452-AEAF-4E28-92D5-CCC5EE9F0A4C",
+    "AccessValidatorCheckRightsServiceURL": "rabbitmq://localhost/CheckRightsService",
+    "AccessValidatorUserServiceURL": "rabbitmq://localhost/UserService"
+	},
```

Comment:
Fix spaces.

Reply:
And again I don't see _any_ reason to fix _anything_.
![image](https://user-images.githubusercontent.com/61093487/91718073-5b279380-eb9b-11ea-9ce7-99e90a58371b.png)